src/Symfony/Contracts/Service/Attribute/SubscribedService.php

```diff
@@ -22,12 +27,21 @@
 #[\Attribute(\Attribute::TARGET_METHOD)]
 final class SubscribedService
 {
+    /** @var object[] */
+    public array $attributes;
+
     /**
-     * @param string|null $key The key to use for the service
-     *                         If null, use "ClassName::methodName"
+     * @param string|null       $key        The key to use for the service
+     * @param class-string|null $type       The service class
+     * @param bool              $nullable   Whether the service is optional
+     * @param object|object[]   $attributes One or more dependency injection attributes to use
```

Comment:
What about accepting only one attribute since we deal with only one?

Reply:
I thought by [this comment](https://github.com/symfony/symfony/pull/47101#discussion_r947000170) you wanted to keep it as an array so it's future-proof?

Reply:
haha ok bad memories :)